### PR TITLE
MUMUP-1986&1985 Change the "Add to favorites" & Already in favorites" button label 

### PR DESCRIPTION
**Pull request by jiayinjx**
Add/Added to home instead of Add to home/Already in favorite

### CODE REVIEW

File: angularjs-portal-frame/src/main/webapp/samples/sidebar.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {"sidebar" :  
             [
                 {"title": "Home", "hyperlink" : "/web", "target" : "", "faIcon" : "fa-home", "displayFlag" : ""},
-                {"title": "Apps", "hyperlink" : "/web/apps", "target" : "", "faIcon" : "fa-star", "displayFlag" : ""},
```
Comment (jiayinjx) [inline angularjs-portal-frame/src/main/webapp/samples/sidebar.json]: change icon and title which is original star and "Apps".

**Comment by timlevett** [comment]
:+1:

**Comment by timlevett** [comment]
I'll fix the merge conflicts

**Comment by jiayinjx** [comment]
Thanks so much!

**Comment by timlevett** [comment]
Merged manually.

**Pull request merged**